uk.co.compucorp.civicrm.hrcomments/tests/phpunit/CRM/HRComments/BAO/CommentTest.php

```diff
@@ -127,4 +150,69 @@ public function testCreatedAtCannotBeChangedWhenUpdatingExistingComment() {
 
     $this->assertEquals($timestampNow, $createdDateTimestamp, '', 10);
   }
+
+  public function testSoftDeleteDoesNotDeleteCommentsFromCommentsTableButSetsIsDeletedFlagToOne() {
+    $comment1 = CommentFabricator::fabricate([
+      'entity_id' => 1,
+      'contact_id' => 1,
+    ]);
+
+    $comment2 = CommentFabricator::fabricate([
+      'entity_id' => 2,
+      'contact_id' => 2,
+    ]);
+
+    Comment::softDelete($comment1->id);
+
+    $comment = new Comment();
+    $comment->find();
+    $this->assertEquals($comment->N, 2);
+
+    $comment->fetch();
+    $this->assertEquals($comment->id, $comment1->id);
+    $this->assertEquals(1, $comment->is_deleted);
+
+    $comment->fetch();
+    $this->assertEquals($comment->id, $comment2->id);
+    $this->assertEquals(0, $comment->is_deleted);
+  }
+
+  public function testIsDeletedAtWillBeZeroIrrespectiveOfTheValuePassedViaIsDeletedParameterOnCreate() {
```

Comment:
The test name says "testIsDeleted**At**" but it should probably be "testIsDeleted"